doc/Language/objects.pod6

```diff
@@ -263,13 +260,13 @@ Method names can be resolved at runtime with the C<.""> operator.
 =head2 self
 
 Inside a method, the term C<self> is available, which is bound to the
-invocant, i.e.  the object that the method was called on. C<self> can be
+invocant, the object that the method was called on. C<self> can be
 used to call further methods on the invocant. Within methods, something like
-C<$.origin> is the same thing as C<self.origin>, however the colon-syntax for
+C<$.origin> is the same thing as C<self.origin>; however, the colon-syntax for
 method arguments is only supported for method calls using self, not the shortcut.
 
-Please note that if the relevant methods C<bless>, C<CREATE>, C<BUILDALL> of
-L<Mu> are not overloaded C<self> will point to the type object in those
+Note that if the relevant methods C<bless>, C<CREATE>, C<BUILDALL> of
```

Comment:
I liked it better with ‘please’, but I think this is just personal opinion. Ultimately, I am fine either way.